src/OrbitTest/OrbitTest.cpp

```diff
@@ -15,16 +15,26 @@
 #include <string>
 #include <thread>
 
-#include "../Orbit.h"
 #include "OrbitBase/ThreadUtils.h"
 #include "absl/strings/str_format.h"
 
+// OrbitBase defines an implementation of the Orbit API for introspection. To prevent symbol clash
+// with the header-only use of the API, we wrap the include of Orbit.h in a namespace. This is only
+// needed for internal projects that link with OrbitBase, basically only OrbitTest.
+namespace orbit_api_wrapper {
+#include "Api/Orbit.h"
+}
```

Comment:
This is quite weird. Is there no alternative? Do we even need the same interface for manual instrumentation and for introspection?

Reply:
Yeah it is definitely weird, but it's only for this one internal test project. Another option would be to not link with OrbitBase, but that's annoying because we're using ThreadPools and Logging. We could duplicate the interface, but then we'd maintain essentially 2 copies of the same file. I also find it cool that we're dogfooding the API.

Reply:
> We could duplicate the interface, but then we'd maintain essentially 2 copies of the same file. I also find it cool that we're dogfooding the API.

I see your points. On the other hand it seems that we have to go through various hacks in order to achieve this (such as this one or `#ifdef`s) to have only one file. And introspection and manual instrumentation work quite differently under the hood. So I'm not really sold on the fact that it's better to have a single "copy".

Reply:
Yeah, I'm not particularly happy with this namespace trick, I'll think about an alternative. Maybe having separate interfaces for introspection and manual instrumentation, we'll see, but as mentioned I think it'll be easier to tackle this once the original manual instrumentation is deprecated.

Reply:
Sounds good. Maybe add a todo (possibly with bug) so that we don't lose track.